### DISCUSSION TITLE
E-Mail Settings

### DISCUSSION THREAD

**Discussion by Instahyper** [bluet/docker-cronicle-docker#31] [Q&A]
Hi,
is there a way to setup third party mail handler?
sometimes i read about the config.json but not for dockerized version

could you pls help me out here?

br frank

**Comment by bluet** [comment]
hi @Instahyper, would you mind letting me know the steps to reproduce? Thanks!

**Comment by bluet** [comment]
I've just merged a PR about custom config https://github.com/bluet/docker-cronicle-docker/pull/39 , maybe it's the cause. I'll upload new docker images later today, please test it and see if that works 😉

**Reply by bluet** [reply]
just uploaded the new image
